mirgecom/initializers.py

```diff
@@ -588,9 +856,24 @@ def __call__(self, t, x_vec, eos=IdealSingleGas()):
         eos: :class:`mirgecom.eos.GasEOS`
             Equation of state class to be used in construction of soln (unused)
         """
-        return _make_uniform_flow(x_vec=x_vec, mass=self._rho,
-                                  pressure=self._p, energy=self._e,
-                                  velocity=self._velocity, eos=eos)
+        gamma = eos.gamma()
+        mass = 0.0 * x_vec[0] + self._rho
+        mom = self._velocity * make_obj_array([mass])
+        energy = (self._p / (gamma - 1.0)) + np.dot(mom, mom) / (2.0 * mass)
+        massfrac = None
+        if self._massfrac is not None:
+            massfrac = self._massfrac * make_obj_array([mass])
+
+        from mirgecom.euler import join_conserved
+        return join_conserved(dim=self._dim, mass=mass, energy=energy,
+                              momentum=mom, massfractions=massfrac)
+    # Save this - some things in Uniform flow generator are broken
+    # and one or the other of the above or below fixes the issue.
+    # =======
+    #         return _make_uniform_flow(x_vec=x_vec, mass=self._rho,
+    #                                  pressure=self._p, energy=self._e,
+    #                                  velocity=self._velocity, eos=eos)
+    # >>>>>>> master
```

Comment:
Explain. What's broken? Why can't we just fix it?

Reply:
This was a temporary note to myself, used during a merge.  Should have been removed.